### PR TITLE
ERD fixes, phase 2

### PR DESCRIPTION
**Pull request by gmile**
Primary changes:

1. removed tables related to auth,
2. added created_at / updated_at for all tables that were missing it.

Also, some misc cleanups here and there.

### CODE REVIEW

File: specs/database_schema/prm.erd [hunk 1]
```diff
@@ -21,15 +21,15 @@ created_at | timestamp | not null
...
 updated_by | uuid | not null
-active | boolean
+is_active | boolean
 scope | enum('') | not null
 division_id | uuid | not null
 
 [declaration_requests]
 id | uuid | not null
-creator_id | varchar | not null
+creator_by | uuid | not null
```
Comment (PavelVesnin) [inline specs/database_schema/prm.erd]: I think it's more consistent rename to `created_by`

File: specs/database_schema/prm.erd [hunk 2]
```diff
@@ -21,15 +21,15 @@ created_at | timestamp | not null
...
 data | jsonb | not null
-ts | timestamp | not null
+created_at | timestamp | not null
```
Comment (PavelVesnin) [inline specs/database_schema/prm.erd]: and everywhere with `created_at` should be `updated_at | timestamp | not null`

File: specs/database_schema/prm.erd [hunk 3]
```diff
@@ -41,7 +41,7 @@ id | uuid | not null
 user_id | varchar | not null
 resource | varchar | not null
 what_changed | json not null
-ts | timestamp | not null
+created_at | timestamp | not null
```
Comment (PavelVesnin) [inline specs/database_schema/prm.erd]: `updated_at | timestamp | not null`

File: specs/database_schema/prm.erd [hunk 4]
```diff
@@ -90,22 +90,22 @@ science_degree | jsonb
 [employee_requests]
 id | int(11) | not null
 data | jsonb | not null
-ts | timestamp | not null
+created_at | timestamp | not null
```
Comment (PavelVesnin) [inline specs/database_schema/prm.erd]: `updated_at | timestamp | not null`

File: specs/database_schema/prm.erd [hunk 5]
```diff
@@ -117,7 +117,7 @@ id | uuid | not null
 user_id | uuid | not null
 resource | varchar | not null
 what_changed | jsonb | not null
-ts | timestamp | not null
+created_at | timestamp | not null
```
Comment (PavelVesnin) [inline specs/database_schema/prm.erd]: `updated_at | timestamp | not null`

File: specs/database_schema/prm.erd (hunk 3 above)
Comment (gmile) [inline specs/database_schema/prm.erd]: Это запись-лог, ее нельзя менять. Тут только created_at должен быть

File: specs/database_schema/prm.erd (hunk 5 above)
Comment (gmile) [inline specs/database_schema/prm.erd]: Это запись-лог, см. мой комент выше

File: specs/database_schema/prm.erd [hunk 6]
```diff
@@ -144,34 +158,15 @@ updated_by | uuid | not null
 id | uuid | not null
 name | varchar | not null
 parameters | jsonb[] | not null
-
-[registration_requests]
```
Comment (gmile) [inline specs/database_schema/prm.erd]: Mr. @AlekzandrV will have these as a separate ERD

**Pull request merged**